src/components/AddCard/AddCard.jsx

```diff
@@ -17,7 +17,7 @@ const propTypes = {
  */
 const AddCard = ({ onClick, title, className }) => (
   <ClickableTile className={classNames('add-card', className)} handleClick={onClick}>
-    <p className="title">{title}</p>
+    <p className="addcard-title">{title}</p>
```

Comment:
We should be using the iot prefix for our unique class names moving forward.

Reply:
Fixed